src/components/panels/details/info-multiple-selection.tsx

```diff
@@ -23,6 +23,7 @@ export default function InfoMultipleSection() {
             </Heading>
             <VStack m="var(--chakra-space-1)">
                 {selected.map(node => {
+                    if (!node.startsWith('stn') && !node.startsWith('misc_node')) return;
```

Comment:
Why can't we link the line details of the selected one? Or is this selection only for nodes?

Reply:
Moving is only for nodes, we cannot move a line.

Reply:
We can not get the attrs from getNodeAttrs(). This is a tentative operation to fix the crash bug. 

I'll make the lines visible in #528.